cephfs/admin/fs_quiesce_reef_test.go

```diff
@@ -0,0 +1,25 @@
+//go:build !squid && ceph_preview
+
+package admin
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFSQuiesce(t *testing.T) {
+	fsa := getFSAdmin(t)
+	volume := "cephfs"
+	group := NoGroup
+	fsa.CreateSubVolume(volume, group, "quiesceMe", nil)
+	defer func() {
+		err := fsa.RemoveSubVolume(volume, group, "quiesceMe")
+		assert.NoError(t, err)
+	}()
+	ret, err := fsa.FSQuiesce(volume, group, []string{"quiesceMe"}, "", nil)
+	assert.Nil(t, ret)
+	var notImplemented NotImplementedError
+	assert.True(t, errors.As(err, &notImplemented))
```

Comment:
Instead of `errors.As()` which places the error converted into `notImplemented`, you can check for the error type with this:
```go
assert.True(t, errors.Is(err, NotImplementedError))
```

Reply:
This is a nice to have, but not required from my POV. I'm ready to approve, but I want to give @nixpanic an opportunity to change this to 'changes requested' if this is highly desires and/or @manishym  a chance to update the code.
Otherwise I plan to approve in 24hrs or so. :-)

Reply:
I tried this, it does not work. 
`→ go test -run TestFSQuiesce -tags reef,ceph_preview
# github.com/ceph/go-ceph/cephfs/admin [github.com/ceph/go-ceph/cephfs/admin.test]
./fs_quiesce_reef_test.go:32:32: NotImplementedError (type) is not an expression
note: module requires Go 1.19
FAIL    github.com/ceph/go-ceph/cephfs/admin [build failed]`
I also tried giving Variable instead of type,
`→ go test -run TestFSQuiesce -tags reef,ceph_preview
--- FAIL: TestFSQuiesce (0.17s)
    fs_quiesce_reef_test.go:25:
                Error Trace:    /home/myathnalli/work/go-ceph/cephfs/admin/fs_quiesce_reef_test.go:25
                Error:          Should be true
                Test:           TestFSQuiesce
FAIL
exit status 1
FAIL    github.com/ceph/go-ceph/cephfs/admin    0.236s`

Reply:
I don't know why that would fail. It would just be a little cleaner, but definitely not a required change that blocks this PR from getting merged.